src/services/item/schemas.packed.ts

```diff
@@ -36,35 +34,62 @@ export const packedItemSchemaRef = registerSchemaAsRef(
       thumbnails: Type.Optional(Type.Object({}, { additionalProperties: true })),
     },
     {
-      // Schema Options
-      additionalProperties: false,
+      description: 'Item with additional information',
```

Comment:
Should we be more precise ?

Reply:
I will add more details on each prop instead.